internal/flow/flow.go

```diff
@@ -46,6 +47,18 @@ type Service struct {
 	NotifyReleaseHook func(ctx context.Context, options NotifyReleaseOptions)
 }
 
+type Storage interface {
```

Comment:
Could be nice with this split into more files, right?

Reply:
Moved into a separate file.